ya: fix args for new updates (#19590)
tldr-pages/tldr: pages/common/ya.md

# ya

> Manage Yazi plugins and flavors, publish or subscribe to DDS messages.
> More information: <https://yazi-rs.github.io/docs/cli>.

- Add a package:

`ya pkg add {{package}}`

- Delete a package:

`ya pkg delete {{package}}`

- Upgrade all packages:

`ya pkg upgrade`

- Emit a command to be executed by the current instance:

`ya emit {{name}} {{args}}`

- Subscribe to messages from all remote instances:

> `ya sub {{kinds}}`

Reviewer: What does kinds mean?

Author: ```
$ ya sub --help    
Subscribe to messages from all remote instances

Usage: ya sub <KINDS>

Arguments:
  <KINDS>  Kind of messages to subscribe to, separated by commas if multiple
```

Reviewer: ```suggestion
`ya sub {{message_kind1,message_kind2,...}}`
```

Reviewer: That's mostly what I do. I document things in the order that I need them.

- Publish a message to the current instance with string body:

> `ya pub --str {{str}}`

Author: No it does not

# ya

> Manage Yazi plugins and flavors, publish or subscribe to DDS messages.
> More information: <https://yazi-rs.github.io/docs/cli>.

- Add a package:

`ya pkg add {{package}}`

- Delete a package:

`ya pkg delete {{package}}`

- Upgrade all packages:

`ya pkg upgrade`

- List all packages:

`ya pkg list`

- Emit a command to be executed by the current instance:

`ya emit {{command}} {{arguments}}`

- Emit commands to a specific remote instance:

> `ya emit-to {{receiver}} {{command}} <arguments>`

Reviewer 2: ```suggestion
`ya emit-to {{receiver}} {{command}} {{arguments}}`
```

- Publish a message to the current instance with string body:

`ya pub --str {{string_message}}`

> - Kind of messages to subscribe to from remote instance:

Reviewer 2: ```suggestion
- Subscribe to a specific kind of message from a remote instance:
```
